### PR TITLE
update basic features and experimental features for v6.2

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/10785
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: experimental-features.md [hunk 1]
```diff
@@ -13,11 +13,16 @@ This document introduces the experimental features of TiDB in different versions
 + [Support collecting statistics for `PREDICATE COLUMNS`](/statistics.md#collect-statistics-on-some-columns) (Introduced in v5.4)
 + [Support synchronously loading statistics](/statistics.md#load-statistics). (Introduced in v5.4)
 + [Control the memory quota for collecting statistics](/statistics.md#the-memory-quota-for-collecting-statistics). (Introduced in v6.1.0)
++ [Use the thread pool to handle read requests from the storage engine](/tiflash/tiflash-configuration.md#configure-the-tiflashtoml-file). (Introduced in v6.2.0)
++ [Cost Model Version 2](/cost-model.md#cost-model-version-2). (Introduced in v6.2.0)
++ [Fast Mode](/develop/dev-guide-read-in-fast-mode.md). (Introduced in v6.2.0)
```
Comment (hfxsd) [inline experimental-features.md:18]:
```suggestion
+ [Fast Scan](/develop/dev-guide-read-in-fast-mode.md). (Introduced in v6.2.0)
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: experimental-features.md [hunk 2]
```diff
@@ -39,13 +41,15 @@ Elastic scheduling feature. It enables the TiDB cluster to dynamically scale out
...
 + TiKV introduces [API V2](/tikv-configuration-file.md#api-version-new-in-v610). (Introduced in v6.1.0)
 
-## Backup and restoration
-
-+ [Back up and restore RawKV](/br/rawkv-backup-and-restore.md) (Introduced in v3.1)
-
 ## Data migration
 
 + [Use WebUI](/dm/dm-webui-guide.md) to manage migration tasks in DM. (Introduced in v6.0)
++ [Configure disk quota](/tidb-lightning/tidb-lightning-physical-import-mode-usage.md#configure-disk-quota-new-in-v620) (Introduced in v6.2.0)
```
Comment (ran-huang) [inline experimental-features.md:47]:
```suggestion
+ [Configure disk quota](/tidb-lightning/tidb-lightning-physical-import-mode-usage.md#configure-disk-quota-new-in-v620) for TiDB Lightning. (Introduced in v6.2.0)
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: experimental-features.md [hunk 3]
```diff
@@ -13,12 +13,15 @@ This document introduces the experimental features of TiDB in different versions
 + [Support collecting statistics for `PREDICATE COLUMNS`](/statistics.md#collect-statistics-on-some-columns) (Introduced in v5.4)
 + [Support synchronously loading statistics](/statistics.md#load-statistics). (Introduced in v5.4)
 + [Control the memory quota for collecting statistics](/statistics.md#the-memory-quota-for-collecting-statistics). (Introduced in v6.1.0)
++ [Use the thread pool to handle read requests from the storage engine](/tiflash/tiflash-configuration.md#configure-the-tiflashtoml-file). (Introduced in v6.2.0)
++ [Cost Model Version 2](/cost-model.md#cost-model-version-2). (Introduced in v6.2.0)
++ [FastScan](/develop/dev-guide-read-in-fast-mode.md). (Introduced in v6.2.0)
```
Comment (hfxsd) [inline experimental-features.md:18]:
```suggestion
+ [FastScan](/develop/dev-guide-use-fastscan.md). (Introduced in v6.2.0)
```

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**